src/ipvs/ip_vs_synproxy.c

```diff
@@ -1186,6 +1256,14 @@ int dp_vs_synproxy_synack_rcv(struct rte_mbuf *mbuf, struct dp_vs_conn *cp,
             *verdict = INET_DROP;
             return 0;
         }
+	
+        /* Window size has been set to zero in the syn-ack packet to Client.
+	 * If get more than one ack packet here, 
+         * it means client has sent a window probe after one RTO.
+         * The probe will be forward to RS and RS will respond a window update.
+         */
+        if (cp->ack_num == 1)
+            syn_proxy_send_window_update(af, mbuf, cp, pp, th);
```

Comment:
1. Is it possible `cp->ack_num` is not 1 when retransmission of client's ack?
2. What if the window update packet gets lost? Could we retransmit the lost packet in this case?